integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/spatial/AbstractSpatialWithinSearchPredicateIT.java

```diff
@@ -172,7 +172,7 @@ protected void initData() {
 		UnsearchableFieldsIndexMapping(IndexSchemaElement root) {
 			geoPoint = root.field(
 					// make the field not searchable
-					"geoPoint", f -> f.asGeoPoint().sortable( Sortable.YES ).projectable( Projectable.YES ).searchable( Searchable.NO )
```

Comment:
Yes, that's weird. We suspect a bug, we hit it somewhere else. I'll report it, it's on my to-do list.

Reply:
FYI it's getting fixed: https://bugs.eclipse.org/bugs/show_bug.cgi?id=547807